src/main/java/org/example/racinggame/controller/RacingCarController.java

```diff
@@ -1,38 +1,40 @@
 package org.example.racinggame.controller;
 
+import java.util.List;
+
 import org.example.racinggame.model.Cars;
 import org.example.racinggame.view.InputView;
 import org.example.racinggame.view.OutputView;
 
 public class RacingCarController {
-
-    private final InputView inputView;
-    private final OutputView outputView;
-
-    public RacingCarController(InputView inputView, OutputView outputView) {
-        this.inputView = inputView;
-        this.outputView = outputView;
+    private final Cars cars;
+    private final String carNames;
+    private final int tryCount;
+    private int currentCount = 0;
+
+    public RacingCarController(String carNames, int tryCount) {
+        this.carNames = carNames;
+        this.tryCount = tryCount;
+        cars = new Cars(carNames);
     }
 
     public void play() {
-        String carNames = inputView.getCarNamesFromUserInput();
-        int tryCount = inputView.getTryCountFromUserInput();
-
-        Cars cars = new Cars(carNames);
+        cars.move();
+        currentCount++;
+    }
 
-        outputView.printResult();
-        for (int i = 0; i < tryCount; i++) {
-            cars.move();
-            printCarPositions(cars);
+    public boolean isEnd() {
+        if (currentCount >= tryCount) {
+            return true;
         }
+        return false;
```

Comment:
아래와 같이 변경해볼 수 있을 것 같아요. 🙂
```suggestion
        return currentCount >= tryCount
```